cloudevents/spec.md

```diff
@@ -124,12 +124,15 @@ Stand-alone event formats, such as the [JSON format](formats/json-format.md),
 specify serialization independent of any protocol or storage medium. Protocol
 Bindings MAY define formats that are dependent on the protocol.
 
+Each Event Format MUST define a structured-mode representation, and MAY define
```

Comment:
It might be me but it feels like the word "transport" needs to be there to make it clear it's a "transport format" and not really an "event format that's independent of a transport". Perhaps just "Each event transport format MUST..." ?? Or even just "Each transport format MUST...."